pom.xml

```diff
@@ -28,7 +28,7 @@
   <artifactId>rhombat</artifactId>
   <name>rhombat</name>
   <packaging>jar</packaging>
-  <version>1.0.2</version>
+  <version>1.0.3-SNAPSHOT</version>
```

Comment:
at what point does the `-SNAPSHOT` get removed? Also, is this a breaking change that should bump the major version?

Reply:
According to everything I know, the version in `develop` should always end in `-SNAPSHOT`. See http://maven.apache.org/maven-release/maven-release-plugin/examples/prepare-release.html and https://gist.github.com/lemiorhan/97b4f827c08aed58a9d8